tests/gocase/unit/introspection/introspection_test.go

```diff
@@ -40,6 +40,24 @@ func TestIntrospection(t *testing.T) {
 	rdb := srv.NewClient()
 	defer func() { require.NoError(t, rdb.Close()) }()
 
+	t.Run("TIME", func(t *testing.T) {
+		now_ms := int(time.Now().Unix())
```

Comment:
@enjoy-binbin Just a little fix - in Go, the snakeCase is preferred, so you can rename the variable to `now`/`nowMs`.
P.S. According to the docs, `time.Now().Unix()` returns seconds not milliseconds. `UnixMilli()` returns milliseconds if you need milliseconds. Otherwise, the variable that holds seconds shouldn't hold seconds.